src/zui/ZUIOrganizeSidebar/index.tsx

```diff
@@ -123,9 +129,32 @@ const ZUIOrganizeSidebar = (): JSX.Element => {
   const treeDataList = useSelector(
     (state: RootState) => state.organizations.treeDataList
   );
+  const currentOrg = useSelector(
+    (state: RootState) => state.organizations.orgData
+  ).data;
```

Comment:
We discussed getting this from the model instead. The model does two things:

1. Initiate a load
2. Return the current information in a future (which will trigger a re-render when the data finishes loading)

Where are you triggering the load currently? It looks to me like it's not happening anywhere, but I'm unable to run the code right now so I'm not sure if that causes problems.

Reply:
I think this is just me misunderstanding, I will change it!

Reply:
This was also in the commit i forgot to push yesterday!